### DISCUSSION TITLE
WW12 - Yaackov

### DISCUSSION THREAD

**Discussion by jacobavitan837** [FPGA-MAFIA/fpga_mafia#589] [Weekly]
# Completed
 - #558 
  - in this issue I learn how to create a test, in this test I implement calculate GCD between tow numbers.
  - I learn to use with `rvc_scanf()` to receive string from the "keyboard" in this test I use in `fpga_mafia\verif\core_rrv\tests\PS2_GCD.vh` to simulate typing in the keyboard.
  - I use in `"string.h"` (by Noam) library to convert a string to a number.
  - I added new tests to the test sequence file ` verif/core_rrv/tb/hw_seq/core_rrv_hw_seq.vh`.
 - #568 
  -I add test to ensure when we create PR (pull request) Fibonacci test will be executed for check.
# WIP 
# Not Started
 - #553

**Comment by amichai-bd** [comment]
@jacobavitan837 - Please add next to/after each issue a short sentence with what was done. (See Noam Weekly as reference)
Edit existing Weekly
And For the next weekly make sure to add it
